helpers/clickClearRefinements.ts

```diff
@@ -1,10 +1,10 @@
 declare namespace WebdriverIOAsync {
   interface Browser {
-    clearRefinements(): Promise<boolean>;
+    clickClearRefinements(): Promise<boolean>;
   }
 }
 
-browser.addCommand('clearRefinements', async () => {
+browser.addCommand('clickClearRefinements', async () => {
   const clearButton = await browser.$(`.ais-ClearRefinements-button`);
```

Comment:
We didn't do any code reviews for previous PRs but this pattern won't work as expected in all cases because there are multiple ClearRefinements widget mounted (one for desktop, one for mobile).

Reply:
Yes, the helpers are meant to only work with one widget of each type for now as it is enough for our current demos and tests.

It is possible to update them to handle multiple widget instances but I don't think this is required for now.